<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapTabWidget</class>
 <widget class="QWidget" name="MapTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,10,0" columnstretch="0,10,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <widget class="QDockWidget" name="blaDockWidget">
     <property name="windowTitle">
      <string>UAV List</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeView" name="blaTreeView">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ControlPanel" name="controlPanelWidget" native="true"/>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="MapWidget" name="mapWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDockWidget" name="spipDD1DockWidget">
     <property name="windowTitle">
      <string>Stations</string>
     </property>
     <widget class="NiippWidget" name="spipDD1Widget"/>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QDockWidget" name="bplaDockWidget">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>93</height>
      </size>
     </property>
     <property name="windowTitle">
      <string>Enemy UAV List</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents_3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeView" name="bplaTreeView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapWidget</class>
   <extends>QWidget</extends>
   <header>Map/MapWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NiippWidget</class>
   <extends>QWidget</extends>
   <header>Niipp/NiippWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControlPanel</class>
   <extends>QWidget</extends>
   <header>ControlPanel/ControlPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
